habitat_baselines/config/default.py

```diff
@@ -30,14 +31,17 @@
 _C.VIDEO_DIR = "video_dir"
 _C.TEST_EPISODE_COUNT = -1
 _C.EVAL_CKPT_PATH_DIR = "data/checkpoints"  # path to ckpt or path to ckpts dir
-_C.NUM_PROCESSES = 16
+_C.NUM_SIMULATORS = 16
```

Comment:
Maybe, `NUM_ENVIRONMENTS` will is more accurate as there is a VecEnv class and Env and Sim wrapper around each simulator.

Reply:
Yeah, I like that way better